data/base/lib/templates/kickstart/domain/manifest.yaml

```diff
@@ -0,0 +1,19 @@
+domain:
+  name: <%= domain.config.cluster %>
+  gateway_ip: <%= domain.config.networks.network1.gateway %>
+groups:
+<% groups.each do |group| -%>
+  - name: <%= group.name %>
+<% end -%>
+nodes: 
+<% nodes.each do |node| -%>
+  - name: <%= node.name %>
+    fqdn: <%= node.config.networks.network1.hostname %>
+    build_ip: <%= node.config.networks.network1.ip %>
+    primary_group: <%= node.group.name %>
+    secondary_groups: <%= "#{node.config.role},#{node.answer.secondary_groups},all".split(',').uniq.reject(&:empty?)  %>
+    kickstart_file: kickstart/node/<%= node.name %>/platform/<%= node.name %>.ks
+    pxelinux_file: kickstart/node/<%= node.name %>/platform/pxelinux.cfg/pxe_bios
+    aws_files: aws/node/<%= node.name %>/platform/<%= node.name %>.yaml
```

Comment:
The `files` should give arrays. Their is some type coercion going on, so not sure what this will do. It safest to define it as:
```
aws_files:
  - aws/node/<%= node.name %>/platform/<%= node.name %>.yaml
```

Reply:
`pxelinux_files` is the same

Reply:
have made aws, azure and pxelinux into lists. Not needed for kickstart?